specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/hyperdrive.json

```diff
@@ -0,0 +1,714 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "HyperDrive REST API",
+    "version": "2019-05-01",
+    "title": "HyperDrive"
+  },
+  "tags": [],
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/hyperdrive/v1.0/{arm_scope}/runs": {
+      "post": {
+        "tags": [
+          "HyperDriveExperiment"
+        ],
+        "summary": "Create an Experiment.",
+        "description": "Create a HyperDrive Experiment.",
+        "operationId": "HyperDrive_CreateExperiment",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ArmScopeParameter"
+          },
+          {
+            "in": "formData",
+            "name": "config",
+            "description": "The configuration file with experiment JSON content. A text file that is a JSON-serialized '#/definitions/HyperDriveCreateExperiment' object.",
+            "type": "file",
+            "required": true
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveExperimentResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a HyperDrive experiment": {
+            "$ref": "./examples/createHyperDriveExperiment.json"
+          }
+        }
+      }
+    },
+    "/hyperdrive/v1.0/{arm_scope}/runs/{run_id}/cancel": {
```

Comment:
Is there a specific reason to use snake_case? I would expect that parameter name to be `runId`

Reply:
For HyperDrive service implemented in python, we dont have camel casing.

Reply:
AutoRest will change the parameter name from camelCase to snake_case as appropriate for client libraries. Unless you are using the swagger to generate service stubs using some other tool, there is no expectation that the placeholder/parameter name in swagger is the same as the parameter in your service.

Reply:
Fixed.